pupil/public/javascripts/mtc-check.js

```diff
@@ -135,20 +135,20 @@
   }
 
   // Provide an indication to the user of how much time they have remaining.
-  GOVUK.updatePageTimer = function () {
-    // Set the page timer to the countdown on page load
-    const pageTimer = document.getElementById('js-page-timer')
-    let questionTimeLimit = GOVUK.getQuestionTimeLimit()
-
-    GOVUK.setTextContent(pageTimer, questionTimeLimit.toString())
-    setInterval(function () {
-      let remaining = Math.ceil((deadline - Date.now()) / 1000)
-      if (remaining < 0) {
-        remaining = 0
-      }
-      GOVUK.setTextContent(pageTimer, remaining.toString())
-    }, 100)
-  }
+  // GOVUK.updatePageTimer = function () {
```

Comment:
I am not personally a big fan of keeping commented code in the file since we can always revert if we need using git